src/manifold/src/impl.h

```diff
@@ -183,8 +183,9 @@ struct Manifold::Impl {
         }
         if (numProp > 0) {
           meshRelation_.triProperties.push_back(
-              ivec3(meshGL.triVerts[3 * i], meshGL.triVerts[3 * i + 1],
-                    meshGL.triVerts[3 * i + 2]));
+              ivec3((uint32_t)meshGL.triVerts[3 * i],
+                    (uint32_t)meshGL.triVerts[3 * i + 1],
+                    (uint32_t)meshGL.triVerts[3 * i + 2]));
```

Comment:
which casting syntax do you prefer?

Reply:
ah, actually `static_cast<uint32_t>`, will change.